### PR TITLE
Solve issue with cards not fitting on smaller screens in Community Ha…

### PR DESCRIPTION
**Pull request by krishkumar84**
Description

This PR fixes https://github.com/layer5io/layer5/issues/5639

Notes for Reviewers
@Ashparshp and @sudhanshutech  kindly review this

**Notes for Reviewers**
https://github.com/layer5io/layer5/pull/5644 its my previous pr and @sudhanshutech want some changes in this to eliminate media queries so i use width and maxwidth property bcz previously size is hardcoded and still and kindly give me the feedback 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 19503ca4449f8a5f5f06ba4e24471f990f3f795f at: https://66acda6943ec4d40b7f78b92--layer5.netlify.app

File: src/components/HandbookCard/HandbookCard.style.js [hunk 1]
```diff
@@ -8,12 +8,16 @@ export const HandbookCardWrapper = styled.div`
...
-	  width: 25rem;
+	  width: 100%;
+	  max-width: 25rem;
 	  box-shadow: 0 5px 10px ${props => props.theme.green00D3A9ToGreyE6E6E6};
 	  overflow: hidden;
 	  height: 19rem;
 	  position: relative;
 		transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+		@media (max-width: 420px) {
+        font-size: 1rem;
```
[the remark this answers is not in the document]
Comment (krishkumar84) [reply src/components/HandbookCard/HandbookCard.style.js]: > Why are you reducing the font-size?

hey can you kindly check the code of my updated commit  that mediaquery is used to reduce the fontsize of card 
image
bcz now it looking like this this text is overflowing thats why i reduce the font size

**Comment by l5io** [comment]
🚀 Preview for commit e50a365764c26d1b426e7dbe80caeb077305468f at: https://66acf1d47275d76da0978676--layer5.netlify.app

**Comment by Ashparshp** [comment]
@krishkumar84 Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by Ashparshp** [comment]
@krishkumar84, is it updated with the recent suggestions from the last meeting? I don't see any commits afterward, so is there anything you would like to update? or is it completed?

**Comment by krishkumar84** [comment]
hi @Ashparshp i update it with suggestion and discuss in meeting and also @sudhanshutech  review this i made changes acc to previous discussion like using the combination of max-width and width and all changes are good that's why i am waiting for it to  merge this pr

**Review by Ashparshp — APPROVED** [review]
LGTM! Thank you @krishkumar84.

**Comment by Ashparshp** [comment]
@sudhanshutech //

**Pull request merged**